translationCoreApps/ScripturePane
# Bugfix #3299/Cleanup defunct code & fix linting errors

#### This pull request addresses:

Cleanup defunct code & fix linting errors

#### How to test this pull request:

test any of the tools.  Should not see any NEW scripture pane errors.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/translationcoreapps/scripturepane/81)
